docs/guide/architecture/component-overview.md

```diff
@@ -9,6 +9,22 @@ The CoAP gateway acts act as a CoAP Client, communicating with IoT devices - CoA
 ![L3](/img/diagrams/component-coapgateway.svg =600x)
 :::
 
+### APIs
+
+CoAP APIs of the Cloud Service are defined in [OCF Device To Cloud Services Specification](https://openconnectivity.org/specs/OCF_Device_To_Cloud_Services_Specification_v2.2.3.pdf).
+
+- POST /oic/sec/account - sign up the device with authorization code
```

Comment:
Move this to architecture, component overview, coapgw.

Reply:
Yes, I already moved that APIs to architecture/component-overview#CoAP Gateway as you see.

Reply:
But you didn't remove it from here :)

Reply:
This part is originally from docs/guide/deployment/coap-gateway.md. So I moved this part to docs/guide/archtecture/component-overview.md#coap-gateway.